.rubocop.yml

```diff
@@ -29,21 +23,9 @@ Layout/IndentHash:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: false
 
-Metrics/ClassLength:
-  Enabled: false
-
-Metrics/LineLength:
-  Enabled: false
-
-Metrics/MethodLength:
-  Enabled: false
-
 Style/AndOr:
   Enabled: false
 
-Style/BlockDelimiters:
```

Comment:
I don't have strong feelings about this one.  I disagree with it -- I like block delimiters to be consistent -- you either use all curly braces or all begin-end delimiters.  https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockDelimiters makes it weird for me.  But not weird enough that I couldn't handle it if others feel strongly.

Reply:
I don't adhere to this one unless I'm checking my code, but it seems like a sensible default.